test/extended/templates/template.go

```diff
@@ -124,7 +125,11 @@ var _ = g.Describe("[sig-devex][Feature:Templates] template-api", func() {
 				t.Errorf("%q: %v", path, err)
 				return
 			}
-			processedList, err := templateprocessingclient.NewDynamicTemplateProcessor(dynamicClient).ProcessToListFromUnstructured(template.(*unstructured.Unstructured))
+			unstructuredTemplate := template.(*unstructured.Unstructured)
+			if len(unstructuredTemplate.GetNamespace()) == 0 {
```

Comment:
this entire commit shouldn't be needed after https://github.com/openshift/library-go/pull/1434